src/main/kotlin/com/cognifide/gradle/aem/instance/LocalInstanceOptions.kt

```diff
@@ -2,16 +2,28 @@ package com.cognifide.gradle.aem.instance
 
 import com.cognifide.gradle.aem.common.AemException
 import com.cognifide.gradle.aem.common.AemExtension
+import com.cognifide.gradle.aem.common.AemTask
 import com.cognifide.gradle.aem.common.file.resolver.FileGroup
 import com.cognifide.gradle.aem.common.file.resolver.FileResolution
 import com.cognifide.gradle.aem.common.file.resolver.FileResolver
 import com.cognifide.gradle.aem.common.file.resolver.Resolver
+import com.fasterxml.jackson.annotation.JsonIgnore
 import java.io.File
+import java.io.Serializable
 
-class LocalInstanceOptions(aem: AemExtension, downloadDir: File) {
+class LocalInstanceOptions(aem: AemExtension) : Serializable {
+
+    private val downloadDir = AemTask.temporaryDir(aem.project, TEMPORARY_DIR)
 
     private val fileResolver = FileResolver(aem, downloadDir).apply { group(GROUP_EXTRA) {} }
 
+    /**
+     * Path in which local AEM instances will be stored.
+     *
+     * Default path is a subfolder named '.aem' under root project directory.
+     */
+    var root: String = aem.props.string("aem.localInstance.root") ?: "${aem.project.rootProject.file(".aem")}"
```

Comment:
@mierzwid idea preserved, only property moved

Reply:
OK

Reply:
Please document this property as well ass overriding of instance options DSL in README.md

Reply:
please raise generic task for documentation 

todo collect all CMD properties and ensuring that all are mentioned in docs

Reply:
docs updated. (minimal effort)